tests/store/test_eulerian_model.py

```diff
@@ -1,16 +1,17 @@
 from helpers import get_eulerian_datapath
+from openghg import standardise
 from openghg.retrieve import search
 from openghg.objectstore import get_bucket
 from openghg.store import EulerianModel
+from openghg.standardise._standardise import standardise
 from xarray import open_dataset
 
 
 def test_read_file():
     test_datapath = get_eulerian_datapath("GEOSChem.SpeciesConc.20150101_0000z_reduced.nc4")
 
     bucket = get_bucket()
-    with EulerianModel(bucket=bucket) as em:
-        proc_results = em.read_file(filepath=test_datapath, model="GEOSChem", species="ch4")
+    proc_results = standardise(bucket=bucket, data_type="eulerian_model", filepath=test_datapath, model="GEOSChem", species="ch4")
```

Comment:
What we should do really is add a `standardise_eulerian` function and test this here. I'm adding this a suggestion though rather than saying this needs to be done in this PR (otherwise we should open an issue).

Reply:
I've now added this function